frontend/public/components/utils/_resource-metrics.scss

```diff
@@ -0,0 +1,24 @@
+.resource-metrics-dashboard {
```

Comment:
BEM means no nesting of block and element.  Are you doing this because you need the additional specificity?  It looks like you shouldn't need to include `.resource-metrics-dashboard` at all.